taskfiles/deps/README.md

```
@@ -0,0 +1,38 @@
+  - The component depends on another via `<lib_name>_ROOT`.
+
+- Build both shared and static libraries when possible.
+
+- Skip unit tests, examples, docs or unused binaries to speed up the install process.
+
+- Prefer disabling lib-specific testing flags (e.g., `CATCH_BUILD_TESTING=OFF`) rather than setting
+  the generic `BUILD_TESTING` to `OFF`.
+
+- **Lastly**, while satisfying the above requirements, remove redundant flags that merely restate
+  default values found in `CMakeLists.txt` and others source CMake files, with two exceptions:
+    - Set `CMAKE_BUILD_TYPE` to `Release` unless another build type is explicitly required.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)